gtests/net/mptcp/unsupported/test_server_1.pkt

```diff
@@ -0,0 +1,15 @@
+
+// connection initiated by packetdrill
+--tolerance_usecs=100000
+`../../common/defaults.sh`
+
+
++0     socket(..., SOCK_STREAM, IPPROTO_MPTCP) = 3
++0     setsockopt(3, SOL_SOCKET, SO_ZEROCOPY, [1], 4) = 0
+
++0     bind(3, ..., ...) = 0
++0     listen(3, 1) = 0
++0	 <  S   0:0(0)                     win 32792  <mss 1460, sackOK, nop, nop, nop, wscale 7, mpcapable v1 flags[flag_h] nokey>
```

Comment:
please fix the alignment like in the other files: one blank line before this one + 2 spaces before `<` to have an indentation compared to the syscalls.